rpms/build.sh

```diff
@@ -40,7 +40,7 @@ fi
 # Building RPM
 $linux rpmbuild --define "_topdir ${rpm_build_dir}" --define "_threads ${threads}" \
         --define "_release ${package_release}" --define "_localstatedir ${directory_base}" \
-        --define "_debugenabled ${debug}" --target ${architecture_target} \
+        --define "_debugenabled ${debug}" --define --target ${architecture_target} \
```

Comment:
There's an empty `--define`. This will prompt an error like this: `error: Macro %__target has empty body`.

Reply:
Done in https://github.com/wazuh/wazuh-packages/pull/156/commits/1bad1fea3ea589e838e3dfe4278575cad7cdd3c8